front-end/src/ui-components/VoteProgress.tsx

```diff
@@ -53,17 +51,20 @@ const VoteProgress = ({ ayeVotes, className, nayVotes, passingThreshold }: Props
 				className={isPassing? 'passing': '' }
 				percent={ayePercent}
 			/>
-			<div
-				id='passingThreshold'
-				style={{ left: passingThresholdPercent + '%' }}
-			>
-				<hr/>
-				<div
-					className={ passingThresholdPercent < 50 ? 'threshold-left' : 'threshold-right'}
+			{
+				// don't show the threshold if it's not been calculated yet
+				thresholdPercent > 0 &&  <div
+					id='passingThreshold'
+					style={{ left: thresholdPercent + '%' }}
 				>
-					Threshold: {bnToStringBalanceDelimitor(passingThreshold)} {tokenSymbol}
+					<hr/>
+					<div
+						className={ thresholdPercent < 50 ? 'threshold-left' : 'threshold-right'}
+					>
+						{isPassing ? 'Failing' : 'Passing'} threshold: {bnToStringBalanceDelimitor(threshold)} {tokenSymbol}
```

Comment:
I wonder if we should replace the token symbol with "votes" (as well as line 44 and 48). So that `token * conviction = votes`. As you can have more votes than actual issued tokens. And then use the token symbols for the votes without conviction.

Reply:
I think this might be equally confusing for users. Because it's not defined anywhere that a vote = KSM * conviction. Although it's quite clear I think that users can vote with their KSM and add a conviction. Adding a new terminology means adding some more help tool tips.

Reply:
This is also true. Removing the token symbol here and adding it to `ReferendumVoteInfo` like you suggested seems like a good solution